Week-2/index2-2.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Static Template</title>
+    <link rel="stylesheet" href="style2-2.css" />
+  </head>
+  <body>
+    <div class="card">
+      <div class="main">
+        <div class="card1"></div>

```

Comment:
why empty div?

Reply:
To make understanding better,I added an empty div mam